salt/modules/scp_mod.py

```diff
@@ -0,0 +1,244 @@
+# -*- coding: utf-8 -*-
+'''
+SCP Module
+==========
+
+.. versionadded:: Fluorine
+
+Module to copy files via `SCP <https://man.openbsd.org/scp>`_
+'''
+from __future__ import absolute_import
```

Comment:
This needs `print_function` and `unicode_literals`.

Reply:
Oh, sorry I always forget them.

Reply:
Added now